### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [OpenSlides/OpenSlides#6240] [General]
Would a Mastodon account for the project be possible?

**Comment by tsiegleauq** [comment]
Sure, but we already successfully neglect our twitter account ;-)

**Reply by linux-lukas** [reply]
Maybe it would be an idea to create a Mastodon account for OpenSlides and then crosspost to Twitter via Crossposter. Many from the Fediverse community might then become aware of OpenSlides, which they would not otherwise do directly.

**Comment by jsangmeister** [comment]
We now have a mastodon account, which is currently more or less active! https://mastodon.social/@OpenSlides@norden.social
